### PR TITLE
MV-7602: Updating CSP Campaign doc

### PR DESCRIPTION
**Pull request by nirmitpatel**
**⚠️Please do not make any changes to spec files in this repository!⚠️**

**All spec changes should be done in the [api-specs](https://github.com/Bandwidth/api-specs) repository.**

**Please follow the [Guide](https://bandwidth-jira.atlassian.net/wiki/spaces/DX/pages/4098359409/How+to+Update+API+Specifications+and+Contribute+to+Developer+Documentation) for contributing to our developer documentation.**

### CODE REVIEW

**Review by ajrice6713 — DISMISSED** [review]

File: site/docs/messaging/campaign-management/csp/campaign-api.mdx [hunk 1]
```diff
@@ -100,13 +100,18 @@ HTTP/1.1 429 Too Many Requests
...
+| `Sample3`                  | No (Required for some usecases)      | 	Message sample. Some campaign tiers require 3 or more message samples. Min length 20. Max length 1024   |
+| `Sample4`                  | No (Required for some usecases)      | 	Message sample. Some campaign tiers require 4 or more message samples. Min length 20. Max length 1024   |
+| `Sample5`                  | No (Required for some usecases)      | 	Message sample. Some campaign tiers require 5 message samples. Min length 20. Max length 1024  |
+| `MessageFlow`              | Yes       | 	Message flow description. Min length 40. Max length 2048 |
+| `HelpMessage`              | Yes       | 	Help message of the campaign. Min length 20. Max length 320  |
+| `HelpKeywords`             | No        | 	Help keywords of the campaign. Max length 320. TCR will default the value to HELP  |
+| `OptinMessage`             | No        | 	Opt-In message of the campaign. Min length 20. Max length 320  |
+| `OptinKeywords`            | No        | 	Opt-In keywords of the campaign. Max length 320.  |
```
Comment (adarkbandwidth) [inline site/docs/messaging/campaign-management/csp/campaign-api.mdx:112]: Opt-In has no default keywords, like OptOutKeywords does?

File: site/docs/messaging/campaign-management/csp/campaign-api.mdx [hunk 2]
```diff
@@ -379,6 +410,13 @@ Location: https://dashboard.bandwidth.com/api/accounts/{accountId}/campaignManag
 		<SubscriberHelp>true</SubscriberHelp>
 		<Sample1>test update sample</Sample1>
 		<AffiliateMarketing>false</AffiliateMarketing>
+		<MessageFlow>This is a sample Message Flow of my campaign.</MessageFlow>
+        <HelpKeywords>HELP</HelpKeywords>
+        <HelpMessage>This is a sample Help Message of my campaign.</HelpMessage>
+        <OptinKeywords>START, YES</OptinKeywords>
+        <OptinMessage>This is a sample Opt-In Message of my campaign.</OptinMessage>
+        <OptoutKeywords>STOP, NO</OptoutKeywords>
+        <OptoutMessage>This is a sample Opt-Out Message of my campaign.</OptoutMessage>
```
Comment (adarkbandwidth) [inline site/docs/messaging/campaign-management/csp/campaign-api.mdx:419]: Indentation?

File: site/docs/messaging/campaign-management/csp/campaign-api.mdx (hunk 1 above)
Comment (nirmitpatel) [reply site/docs/messaging/campaign-management/csp/campaign-api.mdx:112]: Poor design by TCR so unfortunately not. Here's the doc: chrome-extension://efaidnbmnnnibpcajpcglclefindmkaj/https://www.campaignregistry.com/Additional-Data-Capture.pdf?utm_medium=email&_hsmi=230706115&_hsenc=p2ANqtz-9BKBTTyG9_OnYX9s4zE3Irlp46dmCE7A2RRt7jvg5QImweOuSZJV2uYQcGJKZn02etDJUmYe0bfUzzKuiLdDalJO_jmg&utm_content=230706115&utm_source=hs_email

**Review by adarkbandwidth — DISMISSED** [review]

File: site/docs/messaging/campaign-management/csp/campaign-api.mdx (hunk 2 above)
Comment (nirmitpatel) [reply site/docs/messaging/campaign-management/csp/campaign-api.mdx:419]: It just looks weird on the code. If you look at the response payload on http://bw-api-docs-mv-7602.s3-website-us-east-1.amazonaws.com/docs/messaging/campaign-management/csp/campaign-api/#request-url, it's correct. GH is behaving weird for some reason. When they were all aligned, it was looking weird on the actual doc

**Review by ajrice6713 — APPROVED** [review]

**Review by adarkbandwidth — APPROVED** [review]

**Pull request merged**